### DISCUSSION TITLE
setRootPrim when exporting

### DISCUSSION THREAD

**Discussion by sanders3d** [Autodesk/maya-usd#1632] [General] [answered]
When exporting  I would like to option to use "setDeafultRoot" or something similar. At he moment it includes the full hierearchy when you export a geometry.

Ex. if you have "/layout/props/shelf/cannisters/canister_1" and select "shelf" when exporting I would expect that the USD file had "/cannisters/canister_1" but instead it contains the full path.

any ideas?

regards
stefan

**Comment by sanders3d** [comment, marked as answer]
Just found this which better explains what I'm after https://github.com/Autodesk/maya-usd/discussions/1085

**Reply by wtelford** [reply]
Hey Stefan, 
I expect this to land in the MayaUSD 0.12.0 release, which is being QA'd right now.

**Reply by wtelford** [reply]
Just a heads up, 0.12.0 is now available.
https://github.com/Autodesk/maya-usd/releases/tag/v0.12.0

**Reply by sanders3d** [reply]
Worked like a charm. Thanks!
